pkg/plugins/manager/signature/manifest.go

```diff
@@ -194,6 +195,9 @@ func Calculate(ctx context.Context, mlog log.Logger, src plugins.PluginSource, p
 	// Track files missing from the manifest
 	var unsignedFiles []string
 	for _, f := range plugin.FS.Files() {
+		// Ensure slashes are used, because MANIFEST.txt always uses slashes regardless of the filesystem
+		f = filepath.ToSlash(f)
```

Comment:
to mimic the previous behavior, shouldn't this be in LocalFS? (so there is no need to do this for every consumer, even though is just one atm).

https://github.com/grafana/grafana/blob/main/pkg/plugins/localfiles.go#L67

Reply:
I'm not sure about this. I think it should live in the signature check as the conversion is required because of how MANIFEST.txt is implemented (it always uses forward slashes).

If we want to open all files on an NTFS fs and we use `Files()` to get a list of all file paths, we won't be able to open any files if they are returned with a forward slash rather than a backslash.

@wbrowne wdyt?

Reply:
Yeah I think it makes sense to have this as is - this matches old functionality that was missed in the refactor https://github.com/grafana/grafana/blob/v9.2.15/pkg/plugins/manager/signature/manifest.go#L229